from src.utils.exceptions.base_exceptions import BaseException


class DomainException(BaseException):
    pass
